lib/kracken/controllers/authenticatable.rb

```diff
@@ -71,25 +71,13 @@ def check_token_expiry!
       #    delete the cookie
       #
       def handle_user_cache_cookie!
-        if cookies[:_radius_user_cache_key]
-          if cookies[:_radius_user_cache_key] == "none"
-            # Sign out current user
-            session.delete :user_id
+        return redirect_to_sign_in unless session_present?
```

Comment:
Ok, I think I understand why the original code had the wrapping `if cookies[:_radius_user_cache_key]` guard. Not all of our applications require login on every page. And when those pages are setup we are only using `skip_before_actoin :authenticate_user!` on them. The idea being, if there is a user session then we'll handle a cache mismatch, otherwise we'll proceed and let the `authenticated_user!` handle checking if they are signed in and redirecting if they are not.

Basically, that's a lot of words to say, I think this just needs to be:

```suggestion
        return unless session_present?
```